### DISCUSSION TITLE
请问呢，如何自动更新，Upstream Sync怎么添加？

### DISCUSSION THREAD

**Discussion by supwang** [ChatGPTNextWeb/NextChat#781] [Q&A]
请问呢，如何自动更新，Upstream Sync怎么添加？

谢谢！

**Comment by kyrsieger** [comment]
[打开自动更新](https://github.com/Yidadaa/ChatGPT-Next-Web/blob/main/README_CN.md#%E6%89%93%E5%BC%80%E8%87%AA%E5%8A%A8%E6%9B%B4%E6%96%B0)
